src/unclogged/core.clj

```diff
@@ -29,6 +29,26 @@
     "3164" MessageFormat/RFC_3164
     "5424" MessageFormat/RFC_5424))
 
+(defn ^:private ->syslog-msg
+  "Turns defaults + message map into a SyslogMessage."
+  [defaults message]
+  (let [{:keys [message
+                message-id
+                app-name
+                hostname
+                process-id
+                facility
+                severity]}
+        (merge defaults message)]
+    (doto (SyslogMessage.)
```

Comment:
I would use `proxy` here, but it turns out SyslogMessage has a _bit_ of interesting behavior here; specifically `CharArrayWriter` for the `message`.